src/Powercord/apis/settings/store/index.js

```diff
@@ -63,18 +63,34 @@ class SettingsStore extends Flux.Store {
     return Object.keys(this.getSettings(category));
   }
 
+  static pending = [];
```

Comment:
"but I don't know how to fix that."
how about not using Node 12+ features on a Node 10 project